cypress/integration/FileInput/Accepts_file.feature

```diff
@@ -0,0 +1,13 @@
+Feature: The FileInput accepts files
+
+    Scenario: The user provides a file
+        Given a single-file IputField is rendered
+        And the InputField does not hold any files
+        When a file is provided
```

Comment:
Since we're using contains in the Then as well I'd expect `And the InputField does not contain any files`.

And below, I find `When a file is selected` more explicit. To me that describes the action more clearly. But feel free to disagree. (Same comment goes for line 11 and 12 below)